### DISCUSSION TITLE
Compact Letter Display (CLD)

### DISCUSSION THREAD

**Discussion by deedeeharris** [raphaelvallat/pingouin#358] [Q&A] [answered]
Hello,
Very impressive package.
Is there an option to get display the connecting letters report like CLD in r?

Thanks.

**Comment by raphaelvallat** [comment, marked as answer]
Hi @deedeeharris ,

Please see this thread: https://github.com/raphaelvallat/pingouin/issues/205

As explained in the last message, there is an external implementation of CLD that takes Pingouin's output and and return the compact letter display: https://github.com/dalensis/compact-letter-display-for-pingouin-python/blob/main/cld.py

Please note that I have not tried or validated it myself.
